### ISSUE TITLE
Test selectors?

### ISSUE THREAD

**Issue by matthias-margush** [BetterThanTomorrow/calva#1241]
Sorry in advance if this question is covered in the docs and I missed it - I browsed through the docs & source code. 

Is there a way to define test selectors for the test runner? On the command line, for example, I do `clojure -A:dev -X:test :excludes '[:integration]'`. Cider offers some configuration options in emacs: `prompt-for-filters`, `cider-test-default-include-selectors` and `cider-test-default-exclude-selectors`.

Thanks!

**Comment by bpringe** [comment]
I don't believe there is a way to do that currently in Calva. I think a PR would be welcome, though. CC @PEZ

**Comment by PEZ** [comment]
Following CIDER here would be great.
